x-pack/plugins/maps/public/connected_components/timeslider/timeslider.tsx

```diff
@@ -50,6 +61,14 @@ class KeyedTimeslider extends Component<Props, State> {
   private _isMounted: boolean = false;
   private _timeoutId: number | undefined;
   private _subscription: Subscription | undefined;
+  defaultRange: {
```

Comment:
make `private _defaultRange`.

nit: use leading `_` to indicate private variables. It's a convention in the Maps-plugin, so quickly indicate instance state, versus public-interface.